contracts/StablecoinConverter.sol

```diff
@@ -41,7 +41,7 @@ contract StablecoinConverter is EpochTokenLocker {
     mapping(address => Order[]) public orders;
 
     IdToAddressBiMap.Data private registeredTokens;
-    uint public MAX_TOKENS;
+    uint public MAX_TOKENS; // solhint-disable-line
```

Comment:
It is written like a constant and it is supposed to be a constant. But still, we made it a variable, which can be set by the instructor for a specific test case.

Reply:
Is there any particular reason this “constant” is not pre-assigned like in the other contracts?

Reply:
In the real deployment this number will be large (~65k). For unit tests we want to be able to make it small (~2) so that we can test that not more than MAX_TOKENS can be added.

I think it would be ok to use the variable naming convention instead of overriding the sol-linter, but no strong feelings.